modules/user/user.resolver.js

```diff
@@ -16,20 +16,27 @@ const userQuery = {
   getUserByToken: (parent, args, context) => {
     userService.getUser(context.user._id);
   },
-  getUserById: (parent, args, context) => (context.user
-    ? userService.getUser(args.id)
-    : new Error(USER_NOT_AUTHORIZE)),
+  getUserById: (parent, args, context) => {
+    if (context.user) {
+      return userService.getUser(args.id);
+    }
+    return new Error(USER_NOT_AUTHORIZE);
+  },
 };
 const userMutation = {
   registerUser: (parent, args) => userService.registerUser(args.user),
   loginUser: (parent, args) => userService.loginUser(args.user),
   deleteUser: (parent, args) => userService.deleteUser(args.id),
-  updateUserById: (parent, args, context) => (context.user
-    ? userService.updateUserById(args.user, args.id)
-    : new Error('Unauthorized')),
-  updateUserByToken: (parent, args, context) => (context.user
-    ? userService.updateUserByToken(args.user, context.user)
-    : new Error('Unauthorized')),
+  updateUserById: (parent, args, context) => {
+    context.user
```

Comment:
Missing `return` here. This method returns nothing

Reply:
added

Reply:
added